app/app/settings.py

```diff
@@ -74,12 +74,19 @@
     'django.middleware.common.CommonMiddleware',
     'django.middleware.csrf.CsrfViewMiddleware',
     'django.contrib.auth.middleware.AuthenticationMiddleware',
+    'django.contrib.auth.middleware.RemoteUserMiddleware',
     'django.contrib.messages.middleware.MessageMiddleware',
     'django.middleware.clickjacking.XFrameOptionsMiddleware',
     'social_django.middleware.SocialAuthExceptionMiddleware',
     'applicationinsights.django.ApplicationInsightsMiddleware',
+    'custom_auth.CustomSSOMiddleware', 
 ]
 
+# added for custom SSO 
+CREATE_USER = env.bool("CREATE_USER", default=True)
+IS_SUPER = env.bool("IS_SUPER", default=True) 
```

Comment:
Don't we want superuser to be specified in a header so that it can be set on a per request basis?

Reply:
```IS_SUPER``` flag is only set in settings.py  but ```X-AUTH-PROXY-GROUPS``` is sent with every request. If ```X-AUTH-PROXY-GROUPS``` is not sent then ```IS_SUPER``` (if its ```true```) will be used to grand user admin permissions. Look at @c-w comments [^](https://github.com/CatalystCode/doccano/pull/3/files/da3e833f673a980d7f70a5efa3ce402d354ec93f#diff-3f9737c05a00f8739afebff6239e18f4) for details.